safe_transaction_service/history/services/webhooks.py

```diff
@@ -104,6 +108,22 @@ def build_webhook_payload(
                 "txHash": HexBytes(instance.internal_tx.ethereum_tx_id).hex(),
             }
         ]
+    elif sender == SafeMessage:
+        payloads = [
+            {
+                "address": instance.safe,
+                "type": WebHookType.MESSAGE_CREATED.name,
+                "messageHash": instance.message_hash,
+            }
+        ]
+    elif sender == SafeMessageConfirmation:
+        payloads = [
+            {
+                "address": instance.safe_message.safe,  # This could make a db call
+                "type": WebHookType.MESSAGE_CONFIRMATION.name,
+                "messageHash": instance.safe_message.message_hash,
```

Comment:
checking if get address and `message_hash` throw foreign key is doing two queries.

Reply:
The second time is not being evaluated again.